symplyphysics/laws/dynamics/springs/compliance_of_two_serial_springs.py

```diff
@@ -27,7 +31,48 @@
 
 law = Eq(total_compliance, first_compliance + second_compliance)
 
-# TODO: derive law from Hooke's law
+# Derive law from Hooke's law
+## In the case of serial connection, the forces acting on both springs are equal whereas
+## their deformations add up.
+
+# When external force is applied to the first spring, it produces a reaction force on both sides of
+# that spring. That in turn produces a reaction force on both sides of the other spring.
+_external_force = SymSymbol("_external_force")
+_first_spring_reaction = abs(newtons_third_law.law.rhs.subs(newtons_third_law.force_action, _external_force))
+_second_spring_reaction = abs(newtons_third_law.law.rhs.subs(newtons_third_law.force_action, _first_spring_reaction))
+_total_reaction = abs(_external_force)
```

Comment:
Probably should also use 3rd law here: `_total_reaction = abs(newtons_third_law.law.rhs.subs(newtons_third_law.force_action, _second_spring_reaction))`

Reply:
I agree